pyblish_maya/lib.py

```diff
@@ -313,3 +310,28 @@ def _show_no_gui():
 
     messagebox.setStandardButtons(messagebox.Ok)
     messagebox.exec_()
+
+
+class Dock(QtWidgets.QWidget):
+
+    def __init__(self, dock_count, parent=None):
+        super(Dock, self).__init__(parent)
+        QtWidgets.QVBoxLayout(self)
+        object_name = "pyblish_maya.dock" + str(dock_count)
+        self.setObjectName(object_name)
+
+
+def dock(window):
+
+    main_window = None
+    for obj in QtWidgets.qApp.topLevelWidgets():
+        if obj.objectName() == "MayaWindow":
+            main_window = obj
+
+    self._dock_count += 1
```

Comment:
Could indeed, and I've already got a different application that does this.
My question would more be whether this is the responsibility of ```pyblish-maya```? For example ```pyblish-qml``` only ever shows one GUI, although that might be just restrictions on the server/client nature.